packages/access-om3-nuopc/package.py

```diff
@@ -55,9 +55,9 @@ class AccessOm3Nuopc(CMakePackage):
     depends_on("cmake@3.18:", type="build")
     depends_on("mpi")
     depends_on("netcdf-fortran@4.6.0:")
-    depends_on("esmf@8.3.0:")
-    depends_on("fms@2021.03:")
-    depends_on("parallelio@2.5.10:")
+    depends_on("esmf@8.3.0: +debug")
+    depends_on("fms@2021.03: build_type==RelWithDebInfo precision=64 +large_file ~gfs_phys ~openmp ~quad_precision")
```

Comment:
I think the choice of using or not OpenMP should be made consistent with the choice for OM3 itself.

Reply:
Addressed in https://github.com/ACCESS-NRI/spack-packages/pull/103/commits/e5117113afde4a432860e4cad5c4f5a9181ab72b